src/components/member-card/index.js

```diff
@@ -31,7 +31,9 @@ const Card = ({ developerInfo }) => {
       {dev && <SuperUserOptions username={username} />}
       <motion.img
         layoutId={username}
-        src={`${img_url}?${Math.random() * 100}`}
+        src={
+          isMember ? `${img_url}?${Math.random() * 100}` : '/images/Avatar.png'
```

Comment:
this is right but we have to avoid the fetching of images from here as well -  https://github.com/Real-Dev-Squad/website-members/blob/develop/src/pages/index.js#L58

Reply:
actually console errors are not appearing anymore, so why do have to refactor this snippet?

![rds_before_change](https://user-images.githubusercontent.com/23046900/150634548-50f76e38-1f1f-4cfa-8627-b2f7f1c36e16.PNG)
as u can there is no 404 error , check the difference
![rds_avoid_new_members_image_fetch_after_change](https://user-images.githubusercontent.com/23046900/150634497-b83ca9fb-924c-496b-897e-10c5e1214c11.PNG)